redis/src/main/java/org/hibernate/ogm/datastore/redis/RedisDialect.java

```diff
@@ -534,16 +527,16 @@ public JsonEntityStorageStrategy getEntityStorageStrategy() {
 		return entityStorageStrategy;
 	}
 
-	/*
-			 * Construct a key based on the key columns:
-			 * Single key: Use the value as key
-			 * Multiple keys: Serialize the key using a JSON map.
-			 */
+	/**
+	 * Construct a key based on the key columns:
+	 * Single key: Use the value as key
+	 * Multiple keys: Serialize the key using a JSON map.
+	 */
 	private byte[] keyToBytes(String[] columnNames, Object[] columnValues) {
 		if ( columnNames.length == 1 ) {
 
 			if ( columnValues[0] instanceof CharSequence ) {
-				return columnValues[0].toString().getBytes();
+				return toBytes( columnValues[0].toString() );
```

Comment:
That's right. I wanted to point out, that the `if (columnValues[0] instanceof CharSequence)` could be removed since both branches of the if call `toBytes( columnValues[0].toString() );` now.